# Request
---
# Response
bool setclear_error_accepted
